etl/src/main/java/org/pih/hivmigration/etl/sql/VisitMigrator.groovy

```diff
@@ -0,0 +1,42 @@
+package org.pih.hivmigration.etl.sql
+
+class VisitMigrator extends SqlMigrator {
```

Comment:
At a high-level, one question I have is whether we want to do this as a one-off migration, or if this kind of thing can be something that just runs in OpenMRS.  Eg. on a schedule, find all encounters of a type that expect association with a visit and if they are not already associated with a visit, find a matching visit on the same date and location to associate with, and if one is not found, create one and associate it with it.  Have this just always running in the background.

Reply:
Yeah, per our discussion yesterday I think improving our logic to link encounters to visits would be valuable and something we should strongly consider. But I think it's tangential to this PR because I'd likely be in favor of building some sort of scheduled task or other functionality in OpenMRS rather than running the Migrator code at regular intervals.

Reply:
@mogoodrich I agree.  My point was more - if we have a scheduled task that will automatically link encounters to new or existing visits, we don't need this migration at all.